orb-slot-ctrl/src/main.rs

```diff
@@ -192,8 +196,8 @@ fn main() -> eyre::Result<()> {
                 }
             }
         }
-        Commands::GitCommit => {
-            println!("{}", env!("GIT_COMMIT"));
+        Commands::GitDescribe => {
+            println!("{}", BUILD_INFO.git.describe);
```

Comment:
Pointing this out: AFAIK no one relies on this command, please correct me if I am wrong.

Reply:
I personally have no idea 🤔